libs-scala/grpc-server-reflection-client/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
```

Comment:
actually CI is failing because of that since so far everything in `libs-scala` supported 2.13 and it was already part of the CI job. Unless there is a strong reason not to support 2.13 here let’s just do it.

Reply:
You convinced me so much I'll rewrite everything in Scala 3. :trollface:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8438/commits/8e343cf8ef83d4295ce71ab0bb5da243849501ec